code/modules/mob/living/living.dm

```diff
@@ -1333,6 +1333,10 @@
 			return//dont open the mobs inventory if you are picking them up
 	. = ..()
 
+/mob/living/proc/mob_to_item()
```

Comment:
document this

Reply:
What about this function needs clarification?

Reply:
https://github.com/shiptest-ss13/Shiptest/blob/master/.github/AUTODOC_GUIDE.md

Reply:
This article does not contain an answer to my question. What about this function needs documentation?